### PR TITLE
drop unused functions if instances have them

### LINKED ISSUE #8563: Drop PostgreSQL functions `should_update_person_prop` `update_person_props`
After an internal discussion with @yakkomajuri we confirmed we don't need them anymore.

## Environment

- [X] PostHog Cloud
- [X] self-hosted PostHog, version/commit: _please provide_

## Additional context
* see internal [thread](https://posthog.slack.com/archives/C01MM7VT7MG/p1644588968056419)

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

Drops some functions that we don't use and removed the creation code for.

## How did you test this code?

*Please describe.*

Ran this locally.

### CODE REVIEW

**Review by guidoiaquinti — APPROVED** [review]
LGTM, let's make sure this is an online operation

**Comment by yakkomajuri** [comment]
As long as we don't use it anywhere we should be ok right?

https://github.com/PostHog/posthog/search?q=update_person_props

**Comment by guidoiaquinti** [comment]
> As long as we don't use it anywhere we should be ok right?
> 
> https://github.com/PostHog/posthog/search?q=update_person_props

Well if they don't lock and/or delete any resource yes

**Comment by yakkomajuri** [comment]
I believe this is safe, but you're more knowledgeable here than I am. Will fix the merge conflicts and leave the call up to you to merge if ok?

@guidoiaquinti

**Review by guidoiaquinti — APPROVED** [review]

File: posthog/migrations/0210_drop_update_person_functions.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0209_plugin_logs_disabled"),
+    ]
+
+    operations = [
+        migrations.RunSQL("DROP FUNCTION IF EXISTS update_person_props, should_update_person_props"),
```
Comment (guidoiaquinti) [inline posthog/migrations/0210_drop_update_person_functions.py:13]: Nit: I've just realized we made a typo here. The function is `should_update_person_prop`

**Pull request merged**